Fix:

error: implicitly declaring library function 'strncasecmp' with type 'int (const char *, const char *, unsigned long)' [-Werror,-Wimplicit-function-declaration]

https://github.com/php/php-src/commit/1a8df11364292dd3cac2c406a26e4aafaca62d36
--- ext/curl/config.m4.orig	2018-12-04 11:22:53.000000000 -0600
+++ ext/curl/config.m4	2020-12-30 04:11:29.000000000 -0600
@@ -79,6 +79,7 @@
     AC_PROG_CPP
     AC_MSG_CHECKING([for openssl support in libcurl])
     AC_TRY_RUN([
+#include <strings.h>
 #include <curl/curl.h>
 
 int main(int argc, char *argv[])
@@ -106,6 +107,7 @@
    
     AC_MSG_CHECKING([for gnutls support in libcurl])
     AC_TRY_RUN([
+#include <strings.h>
 #include <curl/curl.h>
 
 int main(int argc, char *argv[])
